libs/components/tabs/src/lib/modules/sectioned-form/sectioned-form.component.scss

```diff
@@ -12,12 +12,7 @@
   --sky-override-sectioned-form-tabs-padding: 0;
 }
 
-@include compatMixins.sky-modern-overrides('.sky-sectioned-form') {
-  --sky-override-sectioned-form-content-padding: var(--sky-padding-even-md)
-    var(--sky-padding-even-xl);
-  --sky-override-sectioned-form-tabs-padding: 0;
-  --sky-override-sectioned-form-tabs-only-padding: var(--sky-padding-even-md);
-}
+
```

Comment:
incredibly nitpicky but 3 lines of space is a little odd 😅

Reply:
Fixed